homework/style.css

```diff
@@ -1,3 +1,154 @@
+body {
```

Comment:
The classes version of your app is not responsive:

![image](https://user-images.githubusercontent.com/2788771/61526031-e8506d00-aa19-11e9-909c-e0ac4c183a36.png)

Reply:
I change some of the id name for it to be better and forget to implement in css.
I will look in to it.